### ISSUE TITLE
[LAVA] Query devices before polling

### ISSUE THREAD

**Issue by chaws** [Linaro/squad#1111]
query devices before poll the jobs, devices that are offline put them at the end for jobs to be checked if finished.

[ci: models: avoid polling jobs waiting more than a week on the backend by chaws · Pull Request #1012 · Linaro/squad](https://github.com/Linaro/squad/pull/1012)

Suggested by @roxell

**Comment by mwasilew** [comment]
I don't think this is a good idea or even feasible. There can be multiple devices of the same type. Do you query one or all of them? It might end up causing more traffic than just checking jobs status.
